apps/coscrad-frontend/src/components/resources/spatial-features/spatial-feature-detail.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { getConfig } from '../../../config';
+import {
+    assertElementWithTestIdOnScreen,
+    assertNotFound,
+    renderWithProviders,
+} from '../../../utils/test-utils';
+import { buildMockSuccessfulGETHandler } from '../../../utils/test-utils/buildMockSuccessfulGETHandler';
+import { testContainerComponentErrorHandling } from '../../../utils/test-utils/common-test-cases/test-container-component-error-handling';
+import { setupTestServer } from '../../../utils/test-utils/setupTestServer';
+import { buildMockIndexResponse } from '../../../utils/test-utils/test-data';
+import { SpatialFeatureDetailContainer } from './spatial-feature-detail.container';
+import { buildDummySpatialFeatures } from './test-utils/build-dummy-spatial-features';
+
+const dummySpatialFeatures = buildDummySpatialFeatures();
+
+const referenceSpatialFeatureToFind = dummySpatialFeatures[0];
+
+const { id: idToFind } = referenceSpatialFeatureToFind;
+
+const endpoint = `${getConfig().apiUrl}/Resources/spatialFeatures`;
+
+const act = (idInLocation: string) =>
+    renderWithProviders(
+        <MemoryRouter initialEntries={[`/Resources/Map/${idInLocation}`]}>
```

Comment:
We could have a helper for this piece. It seems we just need the
- resource base route
- DetailContainer

Reply:
Added.